app/src/main/java/com/android/feedme/MainActivity.kt

```diff
@@ -43,6 +45,7 @@ class MainActivity : ComponentActivity() {
               // Navigation host for the app
               val navController = rememberNavController()
               val navigationActions = NavigationActions(navController)
+              val profileViewModel = ProfileViewModel()
```

Comment:
The instantiation has to be modified as the new way, 

Done in another PR and will be fixed when merging.